components/nimbus/src/tests/test_updating.rs

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use crate::error::Result;
+use crate::updating::*;
+use crate::Database;
+
+// This test crashes lmdb for reasons that make no sense, so only run it
+// in the "safe mode" backend.
+#[cfg(feature = "rkv-safe-mode")]
```

Comment:
Aaaah this is why clippy complains - for context:

Clippy runs on multiple rust ["feature" ](https://doc.rust-lang.org/cargo/reference/features.html) configurations:
- It runs against the defaults features
- Against "all features" enabled
- Against "no features" enabled

When the last runs, the test isn't compiled (since `rks-safe-mode` is disabled) and since there is no test compiled, the imports are unused.

A couple of easy ways forward:
- Add a `#![allow(unused_imports)]` at the top of the file, with a short comment
- In the `mod.rs`, right above the `mod test_updating` add a `#[cfg(feature = "rkv-safe-mode")]` and remove the same `cfg` from above the test function itself since it'll be redundant

I'm happy with either 😄 The latter feels a little nicer, but the first feels a bit more explicit 🤷 Regardless whichever you choose works!

Reply:
Thank you so much! I'm going to add the `#!allow` tag so it's clear in the test file what's going on and we can keep the "safe mode" comment